### PR TITLE
add cypress tests

### PR DESCRIPTION
**Pull request by KulakDmitry**

### CODE REVIEW

**Comment by alexanderchernetsky** [comment]
You need to add a script to your package.json file to the scripts section. So anyone can see how to run e2e tests.

File: cypress/integration/homepage.spec.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+/// <reference types="cypress"/>
+
+describe("home page", () => {
+  beforeEach(() => {
+    cy.visit("/");
+  });
+
+  it("should open movie page when click on the movie", () => {
```
Comment (alexanderchernetsky) [inline cypress/integration/homepage.spec.js:8]: This test fails  https://ibb.co/Vv57QcW  The reason is that the response from the API can be different. So you need to mock API response.

**Review by alexanderchernetsky — CHANGES_REQUESTED** [review]
Added 2 comments.

File: cypress/integration/homepage.spec.js (hunk 1 above)
Comment (KulakDmitry) [reply cypress/integration/homepage.spec.js:8]: fixed

**Review by alexanderchernetsky — APPROVED** [review]

**Pull request merged**